Styra/Opa/OpaClient.cs

```diff
@@ -230,6 +53,28 @@ public OpaClient()
     public OpaClient(string serverUrl)
     {
         opa = new OpaApiClient(serverIndex: 0, serverUrl: serverUrl);
+        _logger = new NullLogger<OpaClient>();
+    }
+
+    /// <summary>
+    /// Constructs a default OpaClient, connecting to a server on the default host and port.
+    /// </summary>
+    /// <param name="logger">The ILogger instance to use for this OpaClient.</param>
+    public OpaClient(ILogger<OpaClient> logger)
+    {
+        opa = new OpaApiClient(serverIndex: 0, serverUrl: sdkServerUrl);
+        _logger = logger;
+    }
+
+    /// <summary>
+    /// Constructs a default OpaClient, connecting to a server on the default host and port.
```

Comment:
I think this should be:
```suggestion
    /// Constructs a default OpaClient, connecting to a specified server address.
```